<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel:disabled { color: gray; }
QLineEdit:disabled { color: gray; }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="measuresGroup">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>158</y>
      <width>201</width>
      <height>106</height>
     </rect>
    </property>
    <property name="title">
     <string>Measures</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>20</y>
       <width>195</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="measuresLayout">
      <item>
       <widget class="QCheckBox" name="centroidalCheck">
        <property name="text">
         <string>Centroidal Distance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="nearestCheck">
        <property name="text">
         <string>Nearest Neighbor Distance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hullCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Convex Hull</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="outputGroup">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>156</y>
      <width>183</width>
      <height>165</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QTextEdit" name="outputText">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>20</y>
       <width>175</width>
       <height>139</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="samplingGroup">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>98</y>
      <width>391</width>
      <height>57</height>
     </rect>
    </property>
    <property name="title">
     <string>Sampling</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>21</y>
       <width>383</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="samplingLayout">
      <item>
       <widget class="QComboBox" name="samplingChoice">
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Combinations</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="samplesLabel">
        <property name="text">
         <string>Samples</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="samplesEdit">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="inputGroup">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>0</y>
      <width>391</width>
      <height>95</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>22</y>
       <width>385</width>
       <height>70</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="inputLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="populationLabel">
        <property name="text">
         <string>Population</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="weightsLabel">
        <property name="text">
         <string>Weights</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="populationEdit">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="weightsEdit">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="populationButton">
        <property name="text">
         <string>Choose</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="weightsButton">
        <property name="text">
         <string>Choose</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>280</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Data</string>
    </property>
   </widget>
   <zorder>inputGroup</zorder>
   <zorder>samplingGroup</zorder>
   <zorder>outputGroup</zorder>
   <zorder>measuresGroup</zorder>
   <zorder>generateButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>updateOutputWindow()</slot>
 </slots>
</ui>
